.github/workflows/release-files.yml

```diff
@@ -153,6 +153,8 @@ jobs:
           sha256sum ${{ steps.get-file-base.outputs.FILE_BASE }}.doxygen.zip > ${{ steps.get-file-base.outputs.FILE_BASE }}.sha256sums.txt
           sha256sum ${{ steps.get-file-base.outputs.FILE_BASE }}.tar.gz >> ${{ steps.get-file-base.outputs.FILE_BASE }}.sha256sums.txt
           sha256sum ${{ steps.get-file-base.outputs.FILE_BASE }}.zip >> ${{ steps.get-file-base.outputs.FILE_BASE }}.sha256sums.txt
+          sha256sum hdf5.tar.gz >> hdf5.sha256sums.txt
```

Comment:
The files need a step to download the artifacts
then a separate step to add the sha.
All conditioned on:
        if: ${{ (inputs.use_environ == 'release') }}